core/src/main/java/feign/ReflectiveFeign.java

```diff
@@ -224,6 +224,27 @@ public RequestTemplate create(Object[] argv) {
       return template;
     }
 
+    private Object expandElements(Expander expander, Object value) {
+      if (value instanceof Collection) {
+        return expandCollection(expander, (Collection) value);
+      } else if (value.getClass().isArray()) {
+        return expandCollection(expander, Arrays.asList((Object[])value));
+      }
+      return expander.expand(value);
+    }
+
+    private List<String> expandCollection(Expander expander, Collection value) {
+      List<String> values = new ArrayList<String>();
+      for (Object element : (Collection) value) {
+        if (element==null) {
+          values.add("");
```

Comment:
I disagree. Null is the only sensible placeholder I can think of for "empty but not a string".